aptos-indexer-processors-sdk/sdk/src/common_steps/order_by_version_step.rs

```diff
@@ -0,0 +1,185 @@
+use crate::{
+    traits::{
+        pollable_async_step::PollableAsyncRunType, NamedStep, PollableAsyncStep, Processable,
+    },
+    types::transaction_context::TransactionContext,
+    utils::errors::ProcessorError,
+};
+use ahash::AHashMap;
+use anyhow::Result;
+use async_trait::async_trait;
+use std::time::Duration;
+
+pub struct OrderByVersionStep<Input>
+where
+    Self: Sized + Send + 'static,
+    Input: Send + 'static,
+{
+    pub ordered_versions: Vec<TransactionContext<Input>>,
+    pub unordered_versions: AHashMap<u64, TransactionContext<Input>>,
+    pub expected_next_version: u64,
+    // pub versions: BTreeSet<TransactionContext<Input>>,
```

Comment:
Intended? if so lets add a TODO for future use

Reply:
this was left behind, i'll remove